libcontainer/rootfs_linux.go

```diff
@@ -107,12 +121,12 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		if err := os.MkdirAll(dest, 0755); err != nil {
 			return err
 		}
-		return syscall.Mount(m.Source, dest, m.Device, uintptr(m.Flags), "")
+		return mountPropagate(m.Source, dest, m.Device, m.Flags, "", m.PropagationFlags)
```

Comment:
mountLabel is passed as "context=<label>" mount option. As per the mount man page, it is file system independent option. That means it should be passed in as part of m.Flags and not part of m.Data. This is strange. What am I missing.